internal/scan-manager/scanners/veracode/pom.xml

```diff
@@ -47,6 +47,26 @@
             <artifactId>log4j</artifactId>
             <version>${log4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>${org.springframework.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>${org.springframework.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
Created https://github.com/wso2/security-tools/issues/116 to track removal of these dependencies from the scanner implementations. 

Please create issues for such future improvements that should be done.